tests/integration/test_notes.py

```diff
@@ -1,9 +1,10 @@
 import pytest
 from conftest import is_valid
+from models.notes import NotesModel
 
 
 @pytest.mark.usefixtures("client_class", "empty_test_db")
-class TestCreate:
+class TestNote:
```

Comment:
This should still be `TestCreate`
Let's create a new class for each action. IMO it is easier to read when organized that way.

So we will have `TestCreate` class and `TestDelete` class